packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -0,0 +1,110 @@
+import {Component, h, Prop, State} from '@stencil/core';
+import {Result} from '@coveo/headless';
+import {ResultContext} from '../result-template-decorators';
+import {parseXML} from '../../../utils/utils';
+import {
+  Bindings,
+  BindStateToI18n,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+
+/**
+ * The ResultUri component displays the URI, or path, to access a result.
+ * @part result-printable-uri-list - The result printable uri list
+ * @part result-printable-uri-list-expanded - The expanded result printable uri list
+ * @part result-printable-uri-list-element - A result printable uri list element
+ * @part result-printable-uri-link - A result printable uri clickable link
+ * @part result-printable-uri-list-ellipsis - The clickable ellipsis of a result uri
+ */
+@Component({
+  tag: 'atomic-result-printable-uri',
+  styleUrl: 'atomic-result-printable-uri.pcss',
+  shadow: false,
+})
+export class AtomicResultPrintableUri {
+  @InitializeBindings() public bindings!: Bindings;
+  @ResultContext() private result!: Result;
+  private minNumOfParts = 3;
+
+  @State() listExpanded = false;
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    collapsedUriParts: () => this.bindings.i18n.t('collapsedUriParts'),
+  };
+  @State() error!: Error;
+
+  /**
+   * The maximum number of Uri parts to display, has to be over the minimum of `3` in order to be effective. Putting `0` will disable the ellipsis.
+   */
+  @Prop() maxNumOfParts = 5;
+
+  private renderEllipsis() {
+    return (
+      <li part="result-printable-uri-list-element">
+        <button
+          part="result-printable-uri-list-ellipsis"
+          aria-label={this.strings.collapsedUriParts()}
+          onClick={() => (this.listExpanded = true)}
+        >
+          ...
+        </button>
+      </li>
+    );
+  }
+
+  private get allParents() {
+    return Array.from(
+      parseXML(this.result.raw.parents!).getElementsByTagName('parent')
```

Comment:
Are the uri segments always going to be on `raw.parents` or could the property name need to be configured?

Also, since the properties of the raw object are `unknown`, it would be safer to assert that `parents` has the correct type needed by `parseXML`. If it is a boolean/number, then we should probably return an empty array.

Reply:
It's always parents from what I know. 

Assuming the parents value is of the wrong type, which shouldn't happen, the addition of the try/catch in the util should be enough